stripe-patrons-data/src/main/scala/com/gu/patrons/services/ParameterStoreService.scala

```diff
@@ -60,6 +68,15 @@ class ParameterStoreService(client: AWSSimpleSystemsManagementAsync, stage: Stag
 }
 
 object ParameterStoreService {
+
+  // please update to AWS SDK 2 and use com.gu.aws.CredentialsProvider
+  lazy val CredentialsProviderDEPRECATEDV1 = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider(ProfileName),
+    new InstanceProfileCredentialsProvider(false),
+    new EnvironmentVariableCredentialsProvider(),
+    new EC2ContainerCredentialsProviderWrapper(), // for use with lambda snapstart
+  )
```

Comment:
copied out of module-aws (so that the v1 dependencies don't get pulled in transitively everywhere)